### PR TITLE
Added hungarian translations

### PR DESCRIPTION
**Pull request by lacexd**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,81 +1,71 @@
-version: '3'
+version: "3"
```
Comment (eldadfux) [inline docker-compose.yml]: @lacexd please revert changes in this file, so I can merge your PR

Comment (lacexd) [reply docker-compose.yml]: @eldadfux reverted it, sorry

**Comment by lacexd** [comment]
Resolved the issues

**Comment by eldadfux** [comment]
Thank you!! Amazing addition to the project 🇭🇺!

**Pull request merged**